### PR TITLE
Generate usage & man page for borgfs

### LINKED ISSUE #3287: borgfs still shows 'mount' in options
When running `borgfs --help`, `borgfs mount`, `borgfs`, etc. the short usage still shows that you should put `mount` after borgfs:

    usage: borgfs mount [-h] [--critical] [--error] [--warning] [--info] [--debug]
                        [--debug-topic TOPIC] [-p] [--log-json]
                        [--lock-wait SECONDS] [--show-version] [--show-rc]
                        [--umask M] [--remote-path PATH] [--remote-ratelimit RATE]
                        [--consider-part-files] [--debug-profile FILE] [-f]
                        [-o OPTIONS] [-P PREFIX | -a GLOB] [--sort-by KEYS]
                        [--first N | --last N]
                        REPOSITORY_OR_ARCHIVE MOUNTPOINT

Whereas the first line should just be:

    usage: borgfs [-h] [--critical] [--error] [--warning] [--info] [--debug]

### PR DESCRIPTION
**Pull request by milkey-mouse**
I also fixed the borgfs help text so it no longer says it needs the `mount` subcommand as well.
Fixes #3287 and #3216.

### CODE REVIEW

**Comment by milkey-mouse** [comment]
This has some weird test errors on `py35` but I don't think they're my fault...

File: setup.py [hunk 1]
```diff
@@ -266,7 +269,7 @@ def generate_level(self, prefix, parser, Archiver):
             if self.generate_level(command + " ", parser, Archiver):
                 continue
 
-            with open('docs/usage/%s.rst.inc' % command.replace(" ", "_"), 'w') as doc:
+            with open('docs/usage/%s.rst%s' % (command.replace(" ", "_"), ".inc" if command != "borgfs" else ""), 'w') as doc:
```
Comment (ThomasWaldmann) [inline setup.py]: is that needed? guess we could also have usage/borgfs.rst including the .inc it would generate without that change.

Comment (milkey-mouse) [reply setup.py]: ok, will try that

**Comment by milkey-mouse** [comment]
yes, --help works on every level

**Comment by ThomasWaldmann** [comment]
@enkore any feedback from you?

**Comment by ThomasWaldmann** [comment]
Assuming you have looked at the man page generated and verified it is ok, I guess this is finished.

Can you collapse the fixup commit with the commit it is fixing?

**Pull request merged**